packages/pigment-css-react/src/processors/styled.ts

```diff
@@ -619,7 +619,11 @@ export class StyledProcessor extends BaseProcessor {
         })),
       );
     }
-    delete styleObj.variants;
+    if (styleObj.isProcessed) {
```

Comment:
Should this check instead be for the presence of the key `isProcessed` instead of checking for true?

Reply:
This feels good to me, any reason for checking that way instead of this one?

Reply:
checking for key's presence makes sure that this object is coming from the new processor regardless of the value.

Reply:
The value can only be `true`, it makes more sense to check for `true` than for the key presence.